WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerWrapperViewController.swift

```diff
@@ -40,7 +42,12 @@ final class JetpackBannerWrapperViewController: UIViewController {
         stackView.translatesAutoresizingMaskIntoConstraints = false
 
         view.addSubview(stackView)
-        view.pinSubviewToAllEdges(stackView)
+        NSLayoutConstraint.activate([
```

Comment:
This could've been avoided by using `edgesForExtendedLayout` but it's [not recommended](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621515-edgesforextendedlayout). So, the VC will extend the layout top to bottom via `extendedLayoutIncludesOpaqueBars` and the `UIStackView` will go from top edge to bottom safe area.